packages/e2e_tests/cypress/support/e2e.js

```diff
@@ -1,9 +1,11 @@
 /// <reference types="@shelex/cypress-allure-plugin" />
 // Import commands.js using ES2015 syntax:
+
 import './commands';
 import '@shelex/cypress-allure-plugin';
 import { slowCypressDown } from 'cypress-slow-down';
 slowCypressDown(); // slows down each command by 500ms
```

Comment:
Best to only add this for the tests which need it.

Reply:
Not sure what to do as this happens randomly